src/jdk.zipfs/share/classes/jdk/nio/zipfs/ZipPath.java

```diff
@@ -713,7 +713,7 @@ public void remove() {
             if (type == FileOwnerAttributeView.class)
                 return (V)new ZipPosixFileAttributeView(this,true);
         }
-        throw new UnsupportedOperationException("view <" + type + "> is not supported");
+        return (V) null;
```

Comment:
I assume (V) isn't needed here.

Reply:
No, it is not and can also probably be removed from open/src/java.base/unix/classes/sun/nio/fs/UnixFileSystemProvider.java at some point